pkg/features/kube_features.go

```diff
@@ -330,9 +330,10 @@ const (
 	// Disables the OpenStack Cinder in-tree driver.
 	InTreePluginOpenStackUnregister featuregate.Feature = "InTreePluginOpenStackUnregister"
 
-	// owner: @huffmanca
+	// owner: @huffmanca, @dobsonj
 	// alpha: v1.19
 	// beta: v1.20
+	// GA: v1.23
```

Comment:
Can you also update the API comments to remove the beta designation?

https://github.com/kubernetes/kubernetes/blob/731ae59918970b95ee8fb509d211c81e200725af/pkg/apis/storage/types.go#L286

Reply:
Good catch, updated.